lib/lol_dba/cli.rb

```diff
@@ -31,18 +31,14 @@ def initialize(path, options)
     def start(arg)
       load_application
       select_action(arg)
-    rescue Exception => exception
-      if @options[:debug]
-        warn "Failed: #{exception.class}: #{exception.message}"
-        warn exception.backtrace.map { |trace| "    from #{trace}" }
-      end
```

Comment:
I had to remove that rescue otherwise, the exit exception would be caught.

Any ideas?